components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.endpoint/src/gen/java/org/wso2/carbon/identity/configuration/mgt/endpoint/ResourceApi.java

```diff
@@ -0,0 +1,300 @@
+                                                         @ApiParam(value = "This represents the corresponding attribute value that needs to be added.", required = true) AttributeDTO attribute) {
+
+        return delegate.resourceResourceTypeResourceNamePost(resourceName, resourceType, attribute);
+    }
+
+    @PUT
+    @Path("/{resource-type}/{resource-name}")
+    @Consumes({"application/json"})
+    @Produces({"application/json"})
+    @io.swagger.annotations.ApiOperation(value = "Add or Replace the attribute\n", notes = "This API is used to store or replace the attribute given by the user.\n", response = AttributeDTO.class)
+    @io.swagger.annotations.ApiResponses(value = {
+            @io.swagger.annotations.ApiResponse(code = 200, message = "Ok"),
+
+            @io.swagger.annotations.ApiResponse(code = 400, message = "Bad Request"),
+
+            @io.swagger.annotations.ApiResponse(code = 401, message = "Unauthorized"),
+
+            @io.swagger.annotations.ApiResponse(code = 500, message = "Server Error")})
+
+    public Response resourceResourceTypeResourceNamePut(@ApiParam(value = "This represents the name of the attribute to be added or replaced.", required = true) @PathParam("resource-name") String resourceName,
+                                                        @ApiParam(value = "This represents the type of the attribute to be added or replaced and can either be the name or id.", required = true) @PathParam("resource-type") String resourceType,
+                                                        @ApiParam(value = "This represents the corresponding attribute value that needs to be added or replaced.", required = true) AttributeDTO attribute) {
+
+        return delegate.resourceResourceTypeResourceNamePut(resourceName, resourceType, attribute);
+    }
+
+    @GET
+    @Path("/resource-type/{resource-type-name}")
+    @Consumes({"application/json"})
+    @Produces({"application/json"})
+    @io.swagger.annotations.ApiOperation(value = "Retrieve all the resources for the given resource type.\n", notes = "This API is used to retrieve all the resources for a given resource type.\n", response = ResourcesDTO.class)
+    @io.swagger.annotations.ApiResponses(value = {
+            @io.swagger.annotations.ApiResponse(code = 200, message = "Ok"),
+
+            @io.swagger.annotations.ApiResponse(code = 400, message = "Bad Request"),
+
+            @io.swagger.annotations.ApiResponse(code = 401, message = "Unauthorized"),
+
+            @io.swagger.annotations.ApiResponse(code = 500, message = "Server Error")})
+
+    public Response resourceResourceTypeResourceTypeNameGet(@ApiParam(value = "This represents the name of the resource type to be retrieved.", required = true) @PathParam("resource-type-name") String resourceTypeName) {
+
+        return delegate.resourceResourceTypeResourceTypeNameGet(resourceTypeName);
+    }
+}
+
```

Comment:
remove unnecessary new line

Reply:
Intentionally left due to a generated class.